numpy/lib/tests/test_function_base.py

```diff
@@ -3331,6 +3332,31 @@ def test_keepdims(self):
         assert_equal(np.percentile(d, [1, 7], axis=(0, 3),
                                    keepdims=True).shape, (2, 1, 5, 7, 1))
 
+    @pytest.mark.parametrize('q', [7, [1, 7]])
+    @pytest.mark.parametrize(
+        argnames='axis',
+        argvalues=[
+            None,
+            1,
+            (1,),
+            (0, 1),
+            (~2, ~0),
```

Comment:
It's just a personal preference since I don't like having to think about 1-based indexing when referencing from the back of the array, I'll change it to use the negative sign instead.

Reply:
It's OK either way.  I just don't think I ever saw it before :).

Reply:
Better to stick to common usage, I think! (though it is kind of neat it works)

Reply:
Oh interesting, I thought it was common usage lol. I saw someone using it on stackoverflow a few years ago and thought it was pretty handy!